src/components/explore/AggregationComparisonGraph.svelte

```diff
@@ -1,9 +1,8 @@
 <script>
   import { Axis } from '@graph-paper/guides';
 
-  import { DataGraphic } from '@graph-paper/datagraphic';
-
   import { tooltip as tooltipAction } from '@graph-paper/core/actions';
+  import DataGraphic from '../../graph-paper/datagraphic/DataGraphic.svelte';
```

Comment:
I think we can drop the `graph-paper/` folder here. Or do you feel there's a reason for it?

Reply:
I intend to use this `graph-paper/` folder for all the components that we're going to pull. Wdyt?

Reply:
Okay. I think "graph-paper" doesn't really have meaning inside glam. And it's an empty folder. I think the graph-paper components should all live with the other components inside `src/components` and subfolders there.

Reply:
Makes sense. I'll move DataGraphics to `src/components/` then